README.md

```diff
@@ -1,8 +1,63 @@
 # Previous
-The bot for ~~annoying nextcord members~~ managing some of nextcord's unique tasks.
+The bot managing the Official [Nextcord Discord Server][NEXTCORDSERVER].
+
+# Features
+
+- **Autopaste:**
+    Automatically paste the contents of a file or codeblock to [our paste service][PASTESERVICE]. \
+    (Source: [cogs/autopaste.py][AUTOPASTEPY])
+
+- **Auto thread:**
+    Automatically create a thread for each message sent in the [AUTO_THREAD_CHANNEL_ID] channel. \
+    (Source: [cogs/autothread.py][AUTOTHREADPY])
+
+- **Automatic stars update:**
+    Automatically pull the stars from the [Nextcord][NEXTCORDREPOSITORY] and [Nextcord v3][NEXTCORDREPOSITORYV3] repoitories and update the [STARS_CHANNEL_ID] channel. \
+    (Source: [cogs/stars.py][STARSPY])
+
+- **Documentation:** 
+    Search through the [Discord][DISCORDDOCS] and [Python][PYTHONDOCS] & [Nextcord][NEXTCORDDOCS] documentation. \
+    (Source: [cogs/discorddoc.py ][DDOCSPY] (Discord) | [cogs/docs.py][DOCSPY] (Python & Nextcord))
+
+- **Help System:**
+    An help system using buttons for the [Nextcord server][NEXTCORDSERVER]. \
+    (Source: [cogs/help.py][HELPPY]))	
+
+- **Others:**
+    - Database: simple database powered by [consul.io][CONSUL].
+    (Source: [cogs/database.py][DATABASEPY])
+    - Bot linking: stores which users bot are added to the nextcord server by boosting.
+    (Source: [cogs/botlink.py][BOTLINKPY])
+    - Charinfo: command to get information about a (unicode) character.
+    (Source: [cogs/etc.py][ETCPY])
 
 # Running the bot
-You need a [consul](https://consul.io) instance running.
+1. Run a [consul.io][CONSUL] instance.
+2. Create or fill in the environment variables shown in the [.env.example][ENVFILE] file.
+3. Run the [main.py](./main.py) file to launch the bot.
 
+Any further help regarding setting up the and getting everything working is not provided.
```

Comment:
You said `regarding setting up the`, I think you wanted to say `regarding setting up the bot`.